includes/class-wc-dintero-hp-checkout.php

```diff
@@ -1127,14 +1128,14 @@ function validate(evt) {
 
 		}
 
-		if($sessionExpired || !$order_id){
+		// if($sessionExpired || !$order_id){
```

Comment:
@chaikovskyia: I'm sure this is fine, but could you explain fast why this is commented out? If it should stay like it, please delete.

Reply:
@andersem , this piece needs to be uncommented.

Reply:
:+1: